src/config/config.cc

```diff
@@ -301,6 +301,19 @@ void Config::initFieldValidator() {
          }
          return Status::OK();
        }},
+      {"workers",
+       [](const std::string &k, const std::string &v) -> Status{
```

Comment:
Actually I think maybe there is no necessary to give a validator, since these checks are trivial and already done in `IntField`.